constructor/osxpkg.py

```diff
@@ -21,6 +21,13 @@
 logger = logging.getLogger(__name__)
 
 
+def check_call(args):
+    """
+    Execute a system process and debug the invocation
+    """
+    logger.debug(f"Executing: {' '.join(args)}")
```

Comment:
```suggestion
    logger.debug("Executing: %s", " ".join(args))
```

Reply:
Just for my understanding, is this for consistency with the existing codebase or is there a reason to stay python <3.6 compatible?

Reply:
Consistency and best practices with `logging`.